resources/Materials/Examples/Syntax/GeomInfos.mtlx

```diff
@@ -22,7 +22,7 @@
     <token name="noisemask" type="string" value="txt/noisemask/noisemask.1002.tif"/>
   </geominfo>
   <geominfo name="gi3" collection="col3">
-    <geomprop name="objsize" type="float" value="5.7" unit="foot"/>
+    <geomprop name="objsize" type="float" value="5.7" unittype="distance" unit="foot"/>
```

Comment:
This should not need a "unittype" specified, only the unit.  From the Spec (p.73): "A geomprop may also specify a unittype and unit if appropriate to indicate that the geometric property's value is in that unit; see the Units section above, although typically the <geompropdef> would define the unittype and unit, and a geomprop would only provide an overriding unit if the unit for its value differed from the geompropdef's defined default unit."